monitorix

```diff
@@ -210,7 +210,7 @@ EOF
     })();
  </script>
  <noscript>
-   <p><img src="$piwik_img" style="border:0;" alt=""/></p>
+   <p><img src="$piwik_img" style="order:0;" alt=""/></p>
```

Comment:
Is this a typo?

Reply:
have no idea how it happened but it is definitely mistake